src/js/constants.js

```diff
@@ -46,12 +46,15 @@ export const MediaStateChangeEvents = {
   MEDIA_SEEKABLE: 'mediaseekablechange',
   MEDIA_PREVIEW_IMAGE: 'mediapreviewimagechange',
   MEDIA_PREVIEW_COORDS: 'mediapreviewcoordschange',
-  // MEDIA_CHROME_ATTRIBUTES: 'media-chrome-attributes',
-  // MEDIA_CONTROLLER: 'media-controller',
   MEDIA_LOADING: 'medialoadingchange',
   USER_INACTIVE: 'userinactivechange',
 };
 
+export const UIControllerAttributes = {
```

Comment:
idea(non-blocking): `MediaStateReceiverAttributes`?

Reply:
Oh yeah, that makes sense. 👍